### PR TITLE
new(MessageItem): Add disable title translation prop.

### PR DESCRIPTION
**Pull request by raimondeee**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Adding a new prop that wraps the title in a span that disables translation option from Google.

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/MessageItem/index.tsx [hunk 1]
```diff
@@ -122,6 +126,12 @@ export class MessageItem extends React.Component<Props & WithStylesProps> {
       warning && styles.container_warning,
     );
 
+    const formatedTitle = disableTitleTranslation ? (
```
Comment (milesj) [inline packages/core/src/components/MessageItem/index.tsx]: @stefhatcher Any reason `title` doesn't support nodes? Seems a bit odd to hardcode this.

**Comment by milesj** [comment]
@raimondeee You need another commit or otherwise the release will fail.

**Comment by raimondeee** [comment]
> @raimondeee You need another commit or otherwise the release will fail.

In addition to merging to the master? Or was that resolved by merging?

**Comment by milesj** [comment]
@raimondeee You need another commit for the merge to work correctly.

File: packages/core/src/components/MessageItem/index.tsx (hunk 1 above)
Comment (stefhatcher) [reply packages/core/src/components/MessageItem/index.tsx]: it's used in other places that are string only

Comment (milesj) [reply packages/core/src/components/MessageItem/index.tsx]: 👍

**Comment by raimondeee** [comment]
Added another commit @milesj and merged to master - please let me know if there are any corrections or additional steps I need to take. Thank you both!

**Comment by milesj** [comment]
Looks good, merging!

**Pull request merged**